mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -341,8 +352,14 @@ class LocalLDAModel private[clustering] (
     }
   }
 
-}
+  /** Java-friendly version of [[topicDistributions]] */
+  def topicDistributions(
+      documents: JavaRDD[(java.lang.Long, Vector)]): JavaRDD[(java.lang.Long, Vector)] = {
+    val distributions = topicDistributions(documents.rdd.asInstanceOf[RDD[(Long, Vector)]])
+    distributions.asInstanceOf[JavaRDD[(java.lang.Long, Vector)]]
```

Comment:
Is this type of class casting safe to do in general? I saw it being used in other places so I just copied it.